tests/testsuite/features2.rs

```diff
@@ -2012,7 +2012,7 @@ fn minimal_download() {
     // none
     // Should be the same as `-Zfeatures=all`
     p.cargo("check -Zfeatures=compare")
-        .masquerade_as_nightly_cargo()
+        .masquerade_as_nightly_cargo(&["features=compare"])
```

Comment:
Should this be `always_nightly`?

Reply:
I'm not sure what this should be. [This makes it seem like it shouldn't be nightly forever](https://github.com/rust-lang/cargo/blob/8529eb1c9641a75667d1fccd8025c3fe46c803d7/src/cargo/core/features.rs#L905-L908), or that it should at least be moved to its own command. The correct decision might be to tag it with both I am unsure

Reply:
Either would be fine, this is a permanently unstable option.  I'm fine with leaving it as-is.